Generals/Code/GameEngine/Source/Common/GlobalData.cpp

```diff
@@ -442,6 +442,9 @@ GlobalData* GlobalData::m_theOriginal = NULL;
 
 	{	"CameraAudibleRadius",				INI::parseReal,				NULL,			offsetof( GlobalData, m_cameraAudibleRadius ) },
 	{ "GroupMoveClickToGatherAreaFactor", INI::parseReal,	NULL,			offsetof( GlobalData, m_groupMoveClickToGatherFactor ) },
+
+	{ "EnablePlayerMoneyPerMinute",	INI::parseBool,				NULL,			offsetof( GlobalData, m_enablePlayerMoneyPerMinute ) },
```

Comment:
```cpp
#if !PRESERVE_RETAIL_BEHAVIOR
{ "EnablePlayerMoneyPerMinute",	INI::parseBool,				NULL,			offsetof( GlobalData, m_enablePlayerMoneyPerMinute ) },
#endif
```

Otherwise players can enable this in GameData.ini and play against Retail players (remember, ZH Network Lobby does not check INI mismatch).